admin_manual/maintenance/code-signing.rst

```diff
@@ -0,0 +1,177 @@
+============
+Code Signing
+============
+
+.. sectionauthor:: Lukas Reschke <lukas@owncloud.com>
+
+ownCloud supports code signing for the core releases as well as dedicated applications. Code signing gives our users an
+additional layer of security by ensuring that nobody else than authorized persons can push updates for your applications.
+
+It furthermore ensures that all upgrades have been executed properly. This means that no additional files are left as well
+as all old files have been properly replaced. In the past invalid performed updates where a major source of errors when
+updating ownCloud.
+
+FAQ
+===
+
+Why did ownCloud add Code Signing?
+----------------------------------
+By supporting Code Signing we add another layer of security by ensuring that nobody else than authorized persons can push
+updates for applications. As well we do ensure that people properly updated their ownCloud instances.
+
+Do we lock down ownCloud?
+-------------------------
+The ownCloud project is open-source and will always be that. We do have no intentions to make it harder for people to
+manually adjust their instances. In fact, any fail with the code integrity of ownCloud core while upgrading will not
+prevent the usage of ownCloud. It will barely display a notification to the administrator. For applications that are not
+tagged "Official" the code signing process is purely optional.
+
+Not open-source anymore?
```

Comment:
This, just like »Do we lock down ownCloud?« is a very negative header and not an actually frequently asked question – right? We should rather go with short bullet points:

- mandatory for apps tagged »Official«
- completely optional for other apps
- only enabled when on the stable or production release branch
- manual adjustment will always be possible

Or something like that.